kedro/runner/parallel_runner.py

```diff
@@ -100,8 +100,9 @@ def _run_node_synchronization(  # pylint: disable=too-many-arguments
     conf_logging: Optional[Dict[str, Any]] = None,
 ) -> Node:
     """Run a single `Node` with inputs from and outputs to the `catalog`.
-    A `PluginManager` `hook_manager` instance is created in every subprocess because
-    the `PluginManager` can't be serialised.
+
+    A `PluginManager` instance is created in each subprocess because the
+    `PluginManager` can't be serialised.
```

Comment:
Nit: There's inconsistent 🇺🇸 / 🇬🇧 spelling of "serialis" (to use the stemmed form) throughout this file/repo.

Reply:
As per https://kedro.readthedocs.io/en/stable/contribution/documentation_contributor_guidelines.html#language we should be using British spelling.

Reply:
@MerelTheisenQB Done, but it's a lot of changes. LMK if you want me to split it into a separate PR (is no problem; can just merge these last 3 commits after).

Reply:
I'm fine with having it as part of this PR.